subsys/event_manager/event_manager.c

```diff
@@ -59,11 +61,20 @@ static void event_processor_fn(struct k_work *work)
 
 		trace_event_execution(eh, true);
 		if (IS_ENABLED(CONFIG_DESKTOP_EVENT_MANAGER_SHOW_EVENTS)) {
-			printk("e: %s ", et->name);
-			if (et->print_event) {
-				et->print_event(eh);
+			if (et->log_event) {
+				char event_log_buf[CONFIG_DESKTOP_EVENT_MANAGER_EVENT_LOG_BUF_LEN];
+				size_t pos;
+
+				pos = et->log_event(eh, event_log_buf,
+					CONFIG_DESKTOP_EVENT_MANAGER_EVENT_LOG_BUF_LEN);
+				__ASSERT_NO_MSG(
+				  (pos < CONFIG_DESKTOP_EVENT_MANAGER_EVENT_LOG_BUF_LEN)
+				  && (pos > 0));
+				LOG_INF("e: %s %s", et->name,
+					log_strdup(event_log_buf));
+			} else {
+				LOG_INF("e: %s", et->name);
```

Comment:
`et->name` is a local/stack string, so you should use `log_strdup` here too

Reply:
unnecessary, discussed offline

Reply:
name is in FLASH